File: RukaLjubavi.Api/Services/Implementations/DrzavaService.cs
using AutoMapper;
using RukaLjubavi.Api.Contracts.Dto;
using RukaLjubavi.Api.Contracts.Requests;
using RukaLjubavi.Api.Database;
using RukaLjubavi.Api.Models;
using RukaLjubavi.Api.Services;

namespace RukaLjubavi.Api.Services.Implementations
{
    public class DrzavaService : BaseDataService<Drzava, DrzavaDto, DrzavaInsertRequest>, IDrzavaService
    {
        public DrzavaService(IMapper _mapper, RukaLjubaviDbContext _context) : base(_context, _mapper)
        {

        }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Services/DrzavaService.kt
package com.fit.ba.rukaljubavi.Services

import com.fit.ba.rukaljubavi.Models.Drzava
import retrofit2.Call
import retrofit2.http.GET

interface DrzavaService {
    @GET("Drzava")
    fun getAll(): Call<List<Drzava>>
}